docs/bugs.json

```
@@ -1,4 +1,16 @@
 [
+    {
+        "uid": "SOL-2022-6",
+        "name": "AbiReencodingHeadOverflowWithStaticArrayCleanup",
```

```suggestion
        "name": "AbiReEncodingHeadOverflowWithStaticArrayCleanup",
```
Since the question of typos came up, shouldn't it be like this?

---

``"SOL-2022-2"`` was ``NestedCallataArrayAbiReencodingSizeValidation"``, that's where I took the ``AbiReencoding`` from... that being said, that's exactly the one with the typo, which we're going to change anyways, so...

---

Depends if you insist on putting a hyphen there or not :)
We used the same form in `NestedCallataArrayAbiReencodingSizeValidation` so I think I'd keep it as is for consistency.

> The name is quite a mouthful :)

Most of them are. Hard to keep them short and unambiguous. Unless we want to start doing the bug branding thing. How about "headbleed" :)

---

Auction off bug names? FTXArenaBug